README.md

```diff
@@ -252,7 +252,7 @@ class MyJob < ApplicationJob
 
   # ...
 ```
-- `key` is the only required parameter, and it can be a symbol, a string or a proc that receives the job arguments as parameters and will be used to identify the jobs that need to be limited together. If the proc returns an Active Record record, the key will be built from its class name and `id`.
+- `key` is the only required parameter, and it can be a symbol, a string or a proc that receives the job arguments as parameters and will be used to identify the jobs that need to be limited together. If the proc returns an ActiveRecord record, the key will be built from its class name and `id`.
```

Comment:
I think [Active Record is written separate](https://guides.rubyonrails.org/active_record_basics.html) 🤔 Unless it's the specific Ruby module, but in that case I think I'd have written `ActiveRecord::Base`.

Reply:
Thanks for the review, that makes sense. I wrongly thought it was referred to the module. Corrected.